oak/server/oak_runtime.cc

```diff
@@ -66,36 +64,30 @@ asylo::Status OakRuntime::StartCompletionQueue(std::unique_ptr<grpc::AsyncGeneri
   module_service_ = std::move(service);
   completion_queue_ = std::move(queue);
 
+  // Start the logging pseudo-node thread.
```

Comment:
I am still not convinced about the pseudo-node terminology. Can we call these things just nodes, and qualify them with what they do, e.g. log node, gRPC node, wasm node, etc.? There is nothing special about wasm nodes apart from the fact that they are configured with a wasm module, but presumably other nodes will also be configurable to some extent, e.g. with credentials or encryption keys for outgoing data flows, etc.

Reply:
I'm not insistent on the specific "pseudo-node" term, but I think there's a significant difference between:

 - "Ordinary" nodes, written in Wasm by the application developer and limited to the functionality provided by the TCB &ndash; in particular, only able to communicate by using the TCB's channels
 - The other nodes, written in C++ by the Oak platform developers (and thus forming part of the TCB), with the full functionality of the wider underlying platform available, including the ability to communicate with the outside world (and this last is basically their _raison d'etre_).

"Pseudo-nodes" seemed like a reasonable shorthand terminology to describe the latter: the ordinary nodes communicate with them in the normal way, but they're not subject to all the constraints of an ordinary node.